behave_test_automation/pah_behave/features/login-testcase.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Login view
+
+    Background:
+      Given User opens pah-fm website
+
+    Scenario: User opens pah-fm login view
+       And User sees "username_field"
+       And User sees "password_field"
+       And User sees "Login" button
+       And User sees three flags: polish, english, ukrainian, in the right side, at top of the page
+
+    # Positive scenario (passing valid login and password)
+    Scenario: User logins to pah-fm system
```

Comment:
Why not write that comment as a part of scenario description?
```suggestion
    Scenario: User successfully logs in to pah-fm system with valid credentials
```

Reply:
Changes applied